Marlin/Configuration_adv.h

```diff
@@ -535,6 +535,15 @@
  */
 //#define CALIBRATION_GCODE
 #if ENABLED(CALIBRATION_GCODE)
+
+  #ifndef BACKLASH_COMPENSATION_PIN
+    #define BACKLASH_COMPENSATION_PIN -1 // Override in pins.h or set to -1 to use your Z endstop
+  #endif
+
+  #define BACKLASH_COMPENSATION_PIN_INVERTING false // set to true to invert the backlash compensation nozzle probe
+  //#define PULLDOWN_BACKLASH_COMPENSATION
+  #define PULLUP_BACKLASH_COMPENSATION
+
```

Comment:
Should maybe BACKLASH_COMPENSATION_PIN_INVERTING and PULLUP_BACKLASH_COMPENSATION be inside the #if def? I assume you only want to use these if BACKLASH_COMPENSATION_PIN is set.

Also, maybe PULLUP_BACKLASH_COMPENSATION should be BACKLASH_COMPENSATION_PIN_PULLUP for consistency.

Reply:
Also, why are you calling it BACKLASH_COMPENSATION? The calibration routine does much more than just the backlash measurements. It also does nozzle offsets. How about:

- CALIBRATION_CUBE_PROBE_PIN
- CALIBRATION_CUBE_PROBE_PIN_INVERTING
- CALIBRATION_CUBE_PROBE_PIN_PULLUP

Reply:
Taken on board, making edits as desired.

Reply:
Or, maybe just:
- CALIBRATION_PROBE_PIN
- CALIBRATION_PROBE_PIN_INVERTING
- CALIBRATION_PROBE_PIN_PULLUP